server/src/main/scala/org/elasticmq/server/config/ElasticMQServerConfig.scala

```diff
@@ -105,6 +107,20 @@ class ElasticMQServerConfig(config: Config) extends Logging {
     QueueSorter.sortCreateQueues(unsortedCreateQueues)
   }
 
+  private def resolveQueryName(nameFromConfig: String, isFifo: Boolean): String = {
+    if (isFifo) {
+      ensureLengthSmallerThan(nameFromConfig + ".fifo", MaximumQueueNameLength)
```

Comment:
Should we add the `.fifo` suffix, or is it added automatically? How does the "real" SQS work - does it return an error, when trying to create a fifo queue with the wrong name, or does it add the suffix itself?

Reply:
The "real" SQS will throw an exception if someone will try to create FIFO queue without the ".fifo" suffix.

Right now, if we will try to add ".fifo" suffix to the queue name in the config, server will throw an exception with message "fifo has type OBJECT rather than BOOLEAN" as the structure of the config has been changed. We are already using the "fifo" (as the property of queue) and expect it to be a boolean value.

The approach presented in this PR is based on the assumption, that users using the config file are relying on the already existing `fifo` property to indicate if the queue is FIFO, not on the `.fifo` suffix.

We could change this approach and enable the possibility to create queues with `.fifo` suffix in the config files. But I am not sure if it is correct approach to allow more than one way to achieve the same thing. It may be misleading for the end user if he can use either ".fifo" suffix or the "fifo" property. What do You think about it?

Reply:
So using `name = "xyz.fifo"` will cause an exception being thrown by the config parser?

Reply:
Ah because the queue name is not a value but a key in the config. Ok, let's add the `.fifo` to the queue name automatically, but also mention this in the readme on config (that this is automatically added).